simple-mbed-cloud-client.cpp

```diff
@@ -76,6 +70,8 @@ int SimpleMbedCloudClient::init() {
     // Older versions: workaround to prevent possible deletion of credentials:
     wait(1);
 
+#ifdef MBED_CLOUD_DEV_UPDATE_ID
+
```

Comment:
Why do we need this macro for Production mode?

Reply:
yes, we need this only for dev mode